recipes/r-bdrc/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set version = '1.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-bdrc
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/bdrc_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/bdrc/bdrc_{{ version }}.tar.gz
+  sha256: 2cfb02988afa0a9b9e1cd73c610e81800a5891d56e21deda7479338409a87876
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-ggplot2
+    - r-gridextra
+    - r-rlang
+    - r-scales
+  run:
+    - r-base
+    - r-ggplot2
+    - r-gridextra
+    - r-rlang
+    - r-scales
+
+test:
+  commands:
+    - $R -e "library('bdrc')"           # [not win]
+    - "\"%R%\" -e \"library('bdrc')\""  # [win]
+
+about:
+  home: https://sor16.github.io/bdrc/
+  dev: https://github.com/sor16/bdrc
```

Sorry, should have been `dev_url`

```suggestion
  dev_url: https://github.com/sor16/bdrc
```